<!DOCTYPE html>
<html>
  <head>
    <title>BEHST: genomic set enrichment analysis enhanced through integration of chromatin long-range interactions</title>
    <link rel="author" href="https://plus.google.com/112579936970974005212/">
    <meta name="author" content="Michael M. Hoffman">

    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css" rel="stylesheet">

  
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-48283039-1', 'pmgenomics.ca');
  ga('send', 'pageview');

</script>
</head>
  <body class="container">

    <h1>BEHST: <small> genomic set enrichment analysis enhanced through integration of chromatin long-range interactions</small></h1>

<blockquote>Chicco D, Bi HS, Reimand J, Hoffman MM. 2017. BEHST &ndash; Genomic set enrichment analysis enhanced through integration of chromatin long-range interactions. <i>In preparation</i>.</blockquote>


<!--    <a href="http://biorxiv.org/content/early/2016/12/20/095463"><i>bioRxiv</i> 095463</a>;
    doi: <a href="https://doi.org/10.1101/095463">https://doi.org/10.1101/095463</a>.
  (<a href="umap.bib" target="_blank">BibTeX</a>)-->


<!-- <p class="lead"><strong>The free BEHST software package efficiently associates functional enriched Gene Ontology terms to input genomic regions.</strong> -->
<figure>
    <img src="./BEHSTschematicsketch4AN.png" width="45%" align="right"/>
</figure>

  <h3>The free BEHST software package efficiently associates functional enriched Gene Ontology terms to input genomic regions</h3>
BEHST reads a dataset of genomic regions, and intersects them with the chromatin interactions available in the Hi-C dataset (<a href="https://doi.org/10.1016/j.cell.2014.11.021">Rao et al, <i>Cell</i>, 2014</a>). Of these genomic regions, BEHST selects those that are present in the regulatory regions of genes a dataset of principal isoform annotations. We defined these <i>cis</i>-regulatory regions upon the position of their nearest transcription start site of the genes' principal transcripts, plus an upstream and downstream extension. Afterwards, BEHST selects the genes of the resulting partner loci found in gene regulatory regions, and inserts them into <a href="http://biit.cs.ut.ee/gprofiler/">g:Profiler</a>. BEHST, finally, produces the list of the most significant Gene Ontology terms detected by g:Profiler.

<h3>Installation</h3>
<p>
BEHST can run on any Linux and Mac computers. You can find the installation instructions on <a href="https://bitbucket.org/hoffmanlab/behst/" target="_blank">BEHST Bitbucket webpage</a>.


<h3>Support</h3>

<p>
For support of BEHST, please write to the <a href="behst-l@listserv.utoronto.ca">BEHST-users mailing list</a>, rather than writing the authors directly. Using the mailing list will get your question answered more quickly. It also allows us to pool knowledge and reduce getting the same inquiries over and over. Questions sent to the mailing list will receive a higher priority than those sent to us individually.
<p>
If you do not want to read discussions about other people's use of BEHST, but would like to hear about new releases and other important information, please subscribe to the <a href="behst-announce-l@listserv.utoronto.ca">BEHST-announce mailing list</a> mailing list. 

<h3>Source code</h3>

<ul>
    <li>
        <a href="https://bitbucket.org/hoffmanlab/behst/" target="_blank">BEHST code repository on Bitbucket</a>
    </li>
</ul>

<h3>Credits</h3>
<p>BEHST was developed by Davide Chicco, Haixin Sarah Bi, Juri Reimand, and Michael M. Hoffman at the <a href="http://www.hoffmanlab.org">Hoffman Lab</a> of the Princess Margaret Cancer Centre (Toronto, Ontario, Canada).
</p>

</script></address>
</body>
</html>
